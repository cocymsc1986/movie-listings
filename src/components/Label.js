import React from 'react';

const Label = ({ label }) => {
	return (
		<div className="movies__genre-label">{label}</div>
	)
};

export default Label;