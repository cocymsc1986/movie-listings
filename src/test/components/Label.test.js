import React from 'react';

import Label from '../../components/Label';

describe('Label component', () => {
	test('Should render', () => {
		const wrapper = shallow(<Label label={"test-label"} />);

		expect(wrapper).toMatchSnapshot();
	});
});
