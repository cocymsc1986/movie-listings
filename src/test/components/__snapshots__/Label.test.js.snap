// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label component Should render 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Label
    label="test-label"
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "test-label",
      "className": "movies__genre-label",
    },
    "ref": null,
    "rendered": "test-label",
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "test-label",
        "className": "movies__genre-label",
      },
      "ref": null,
      "rendered": "test-label",
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getSnapshotBeforeUpdate": true,
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
